google/resource_bigquery_table.go

```diff
@@ -92,6 +92,23 @@ func resourceBigQueryTable() *schema.Resource {
 				},
 			},
 
+			// View: [Experimental] If specified, configures this table as a view.
+			"view": &schema.Schema{
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
```

Comment:
Do you think it makes sense to add the other `view` attributes, or do they not make sense in terraform? (useLegacySql, userDefinedFunctionResources)